.pipelines/singletenancy/aks-swift/e2e-job-template.yaml

```diff
@@ -18,7 +18,6 @@ stages:
           - Role -equals $(CUSTOM_E2E_ROLE)
         container:
           image: "$(BUILD_IMAGE)" # build image set as variable in pipeline runtime for flexibilit
-
```

Comment:
Line 24 here, can you change `unit_tests` to `setup` too please?

Reply:
good catch